src/main/java/frc/robot/Robot.java

```diff
@@ -21,8 +28,11 @@
    * This function is run when the robot is first started up and should be used
    * for any initialization code.
    */
+
+   Limelight limelight;
   @Override
   public void robotInit() {
+    limelight = new Limelight();
```

Comment:
During robot initialization we usually have a good chance of encountering an error (like the limelight not being available because it's not plugged in after a change, etc.). It'll be easier to debug if we have some print statements marking each initialization. For example:

```java
System.out.print("Initializing vision system...");
limelight = new Limelight();
System.out.println("done");
```